Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -34,6 +36,9 @@ private void InitializeShuttle()
     #region Console
     private void UpdatePalletConsoleInterface(EntityUid uid)
     {
+        //Block from overwriting pirate functionality
+        if (TryComp<PiratePalletConsoleComponent>(uid, out _))
+            return;
```

Comment:
This seems like another "why reuse something if we're not reusing all of what it does?"